### PR TITLE
Fonts, include Discord style

### PR DESCRIPTION
**Pull request by Ranzha**

### CODE REVIEW

File: public/css/theme.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 body {
     background-color: #0A0A0A;
     color: #E9E9E9;
+    font-family: Sans-serif, serif;
```
Comment (0az) [inline public/css/theme.css]:
```suggestion
    font-family: sans-serif, serif;
```
Lower case for consistency.

File: public/css/theme.css [hunk 2]
```diff
@@ -19,6 +20,8 @@ body {
 
 .title {
     text-align: center;
+    font-family: Sans-serif, serif;
```
Comment (0az) [inline public/css/theme.css]:
```suggestion
    font-family: sans-serif, serif;
```

File: public/css/theme.css [hunk 3]
```diff
@@ -36,6 +39,14 @@ body {
     text-align: center;
 }
 
+.discord {
+    background-color: #7289DA;
+    color: white;
+    font-family: Sans-serif, serif;
```
Comment (0az) [inline public/css/theme.css]:
```suggestion
    font-family: sans-serif, serif;
```

File: public/css/theme.css (hunk 1 above)
Comment (jacksonargo) [reply public/css/theme.css]: Has to be captialized

**Pull request merged**